### PR TITLE
Update overview.md

### PR DESCRIPTION
**Pull request by jonvnadelberg**
change verify to associates in one instance for consistency.

Partial fix for #262 
#### Release Note

NONE
#### Documentation

Doc only for #262

### CODE REVIEW

**Comment by jonvnadelberg** [comment]
> What do you mean by associated? Linked to an identity?

Yes, this was the wording we settled on in a certain case instead of verified.   You didn't want to use the word verified in a certain instance so we picked "associated."   I thought the other uses of verified were ok, though, so I left them.   Do you want to go back to verified?   I can replace associated with verified.

File: content/en/about/overview.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+rer---
```
[the remark this answers is not in the document]
Comment (jonvnadelberg) [reply content/en/about/overview.md:1]: no.   I'll remove it.

**Comment by haydentherapper** [comment]
I might rewrite that line, using single words rather than phrases doesn't convey much. What about something like:

"With Sigstore, the artifact is not just signed; it’s signed with an ephemeral key, associated with a known identity, and publicly auditable."

**Comment by jonvnadelberg** [comment]
> I might rewrite that line, using single words rather than phrases doesn't convey much. What about something like:
> 
> "With Sigstore, the artifact is not just signed; it’s signed with an ephemeral key, associated with a known identity, and publicly auditable."

That sounds great.   I'll change it.

**Review by cpanato — APPROVED** [review]

**Review by ltagliaferri — APPROVED** [review]

**Pull request merged**